README.md

```diff
@@ -13,13 +13,13 @@ This repository contains the latest version of Human-GEM, a human genome-scale m
 
 If you use Human1 in your research, please cite:  
 
- > J. L. Robinson, P. Kocabas�, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
+ > J. L. Robinson, P. Kocabas�, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
```

Comment:
There seems to be a mishap in the formatting of Pinar's name, so my suggestion is to revert it:
```suggestion
 > J. L. Robinson, P. Kocabasÿ, H. Wang, P.-E. Cholley, et al. An atlas of human metabolism. _Sci. Signal._ 13, eaaz1482 (2020). [doi:10.1126/scisignal.aaz1482](https://doi.org/10.1126/scisignal.aaz1482)
```

Reply:
I already committed the suggestion, but later thought maybe we should stick to the same formatting as in the published article, i.e. "Kocabas". What do others think?

Reply:
Alright, with @haowang-bioinfo's thumbs-up, the formatting has been adjusted in [8cffe89](https://github.com/SysBioChalmers/Human-GEM/pull/647/commits/8cffe891da07cae9cccf0b985eb1283da8537c79).